### PR TITLE
refactor: fix lint errors for line length linter

### PR DESCRIPTION
**Pull request by AlisonB319**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Comment by AlisonB319** [comment]
@alexeldeib @cameronmeissner Here is PR1

File: pkg/agent/baker.go [hunk 1]
```diff
@@ -268,12 +268,20 @@ func normalizeResourceGroupNameForLabel(resourceGroupName string) string {
 func validateAndSetLinuxNodeBootstrappingConfiguration(config *datamodel.NodeBootstrappingConfiguration) {
 	// If using kubelet config file, disable DynamicKubeletConfig feature gate and remove dynamic-config-dir
 	// we should only allow users to configure from API (20201101 and later)
+	dockerShimFlags := []string{"--cni-bin-dir",
+		"--cni-cache-dir",
+		"--cni-conf-dir",
+		"--docker-endpoint",
+		"--image-pull-progress-deadline",
+		"--network-plugin",
+		"--network-plugin-mtu"}
```
Comment (alexeldeib) [inline pkg/agent/baker.go:277]:
```suggestion
	dockerShimFlags := []string{
	    "--cni-bin-dir",
		"--cni-cache-dir",
		"--cni-conf-dir",
		"--docker-endpoint",
		"--image-pull-progress-deadline",
		"--network-plugin",
		"--network-plugin-mtu",
	}
```

nit

File: pkg/agent/baker.go [hunk 2]
```diff
@@ -268,12 +268,20 @@ func normalizeResourceGroupNameForLabel(resourceGroupName string) string {
...
 	profile := config.AgentPoolProfile
 	if config.KubeletConfig != nil {
 		kubeletFlags := config.KubeletConfig
 		delete(kubeletFlags, "--dynamic-config-dir")
 		delete(kubeletFlags, "--non-masquerade-cidr")
-		if profile != nil && profile.KubernetesConfig != nil && profile.KubernetesConfig.ContainerRuntime != "" && profile.KubernetesConfig.ContainerRuntime == "containerd" {
+		if profile != nil && profile.KubernetesConfig != nil && profile.KubernetesConfig.ContainerRuntime != "" &&
```
Comment (alexeldeib) [inline pkg/agent/baker.go:283]: nit: should we do these each line by line? even in the non-diff view, this is not super readable (it wasn't before either, just noticing)

File: pkg/agent/baker.go [hunk 3]
```diff
@@ -268,12 +268,22 @@ func normalizeResourceGroupNameForLabel(resourceGroupName string) string {
...
 	// If using kubelet config file, disable DynamicKubeletConfig feature gate and remove dynamic-config-dir
 	// we should only allow users to configure from API (20201101 and later)
+	dockerShimFlags := []string{
+		"--cni-bin-dir",
+		"--cni-cache-dir",
+		"--cni-conf-dir",
+		"--docker-endpoint",
+		"--image-pull-progress-deadline",
+		"--network-plugin",
+		"--network-plugin-mtu"}
```
Comment (alexeldeib) [inline pkg/agent/baker.go:278]:
```suggestion
		"--network-plugin-mtu",
	}
```

nit: lost one?

**Review by alexeldeib — APPROVED** [review]
one minor comment, I think lgtm -- baker_test hard to read in diff but looks okay on your branch

**Pull request merged**